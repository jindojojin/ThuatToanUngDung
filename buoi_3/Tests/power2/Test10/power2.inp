16
1 5 5 3 3 3 3 7 1111 4 5 2222 5 64 32 12345
