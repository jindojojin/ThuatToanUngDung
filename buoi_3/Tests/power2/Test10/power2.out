1 4 32 64
